### ISSUE TITLE
Update to the latest version of meow

### ISSUE THREAD

**Issue by AlexGustafsson** [xojs/xo#471]
Since May 1st `yargs-parser` has been reported as vulnerable via `npm audit`. It has been fixed in the latest version of `meow`, but that version is in turn not used by XO.

To fix this issue of XO being reported as vulnerable, `meow` will need to be updated to at least `6.1.1`.

I've tried to see if there's any similar issue already opened, but failed to find any. So please forgive me if this is a duplicate.

Output of `npm audit`:

```
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ yargs-parser                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=13.1.2 <14.0.0 || >=15.0.1 <16.0.0 || >=18.1.2             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ xo [dev]                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ xo > meow > yargs-parser                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1500 
```

**Comment by sindresorhus** [comment]
It neither affects Meow nor XO.

**Comment by AlexGustafsson** [comment]
According to the report it does not affect Meow anymore as it's using the latest version of `yargs-parser`. XO however does not as it's using an old version of Meow. I get that it might not directly affect it, but for as long as XO does not update Meow, NPM will warn users of XO about this vulnerability.

XO uses Meow ^5.0.0 (resolves to 5.0.0) which in turn uses yargs-parser ^10.0.0 (resolves to 10.1.0) which is on the affected list: https://www.npmjs.com/advisories/1500/versions.
